datanode/src/main/java/com/odysseusinc/arachne/datanode/environment/EnvironmentDto.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 Odysseus Data Services, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.odysseusinc.arachne.datanode.environment;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import java.util.List;
+
+@Getter
+@RequiredArgsConstructor(staticName = "of")
+public class EnvironmentDto {
```

Comment:
We didn't agree upon this mixture

Reply:
This is just a DTO class for the frontend, nothing more.
As we add docker support, some sort of abstraction above docker and tarball environment is required and that's what this class represents.
If you have ideas how and why it should be done differently, please describe them.